ghost/core/core/server/models/role-utils.js

```
@@ -0,0 +1,41 @@
+// check if the user has an assigned role
+// so that we can stop writing this everywhere:
+//_.some(loadedPermissions.user.roles, {name: 'Administrator'})
+
+const { is } = require('cheerio/lib/api/traversing');
```

ESLint

[error] There should be no space after '{'.

(object-curly-spacing)